source-code/marketplace/buildRegistry.js

```diff
@@ -0,0 +1,96 @@
+import fs from "node:fs/promises"
+
+/**
+ * @typedef {Object} ModuleMetaData
+ * @property {string} id
+ * @property {Record<string, string>} displayName
+ * @property {Record<string, string>} description
+ * @property {Record<string, string>} marketplace
+ * @property {string[]} keywords
+ * @property {string} icon
+ * @property {string} linkToReadme
+ * @property {string} publisherName
+ * @property {string} publisherIcon
+ */
+
+async function main() {
+	const registry = JSON.parse(await fs.readFile("./registry.json", "utf-8"))
+
+	for (let module of registry.modules) {
+		if (module.includes("@latest") || !module.includes("jsdelivr")) continue
```

Comment:
ah, that was because of the files coming from jsDelivr: 
> "modules": [
    "https://cdn.jsdelivr.net/npm/@inlang/plugin-json@3/dist/index.js".
    // ...
  ]

(https://github.com/inlang/inlang/issues/1224#issuecomment-1680101923)

To check whether it's the latest version or not. Is there a better way to check that? Right now the function is just adjusted to return the link with @latest if it is not already stated in the link itself. That's the reason.

Reply:
We should enforce that a module in the registry contains no version number. The marketplace installs the latest major version. Hence, I propose to throw an error if a module link contains a version number